packages/xo-server/src/xo-mixins/proxies.mjs

```diff
@@ -191,6 +194,7 @@ export default class Proxy {
     await xapi._waitObjectState(vmUuid, vm => extractIpFromVmNetworks(vm.$guest_metrics?.networks) !== undefined)
   }
 
+  @decorateWith(debounceWithKey, DEBOUNCE_TIME_PROXY_STATE, id => id)
```

Comment:
It should be shorter or the cache should be invalidated after upgrading a proxy.